RELEASE_NOTES.rst

```diff
@@ -21,6 +21,51 @@
 
 .. towncrier release notes start
 
+Airflow 2.8.3 (2024-03-11)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+The smtp provider is now pre-installed when you install Airflow. (#37713)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Bug Fixes
+"""""""""
+- Add "MENU" permission in auth manager (#37881)
+- Fix external_executor_id being overwritten (#37784)
+- Make more MappedOperator members modifiable (#37828)
+- Set parsing context dag_id in dag test command (#37606)
+
+Miscellaneous
+"""""""""""""
+- Remove useless methods from security manager (#37889)
+- Upgrade google-ads version (#37787)
+- Improve code coverage for TriggerRuleDep (#37680)
+- Limit ``pandas`` to ``<2.2`` (#37748)
+- The SMTP provider is now preinstalled when installing Airflow (#37713)
+- Bump min versions of openapi validators (#37691)
+- Bump lower-limit for pinotdb (#37690)
+- Properly include ``airflow_pre_installed_providers.txt`` artifact (#37679)
+- Remove ``gosu`` binary from our images (#37677)
```

Comment:
This isn't a core change, it should go here: https://airflow.apache.org/docs/docker-stack/changelog.html